docs/pages/project/contributing/meshery-server.md

```diff
@@ -104,3 +104,134 @@ github.com/layer5io/meshkit => ../meshkit
   </li>
 </ol>
 
+### Practicing Schema driven development for Meshery server.
+The guide covers instructions on how to generate Go code from OpenAPI specifications, best practices to ensure the generated code meets the requirements, follows the conventions of the project and necessary pre/post-requisites.
+
+### Installating required tools
+1. `oapi-codegen`: https://github.com/oapi-codegen/oapi-codegen?tab=readme-ov-file#install
```

Comment:
@MUzairS15 describe each of these tools.

Reply:
@MUzairS15 combine the "Install Required Tools" with the "Prerequisites" section below. This should be a step within that section.